fastlane/lib/fastlane/actions/trainer.rb

```diff
@@ -7,6 +7,8 @@ def self.run(params)
         params[:path] = Actions.lane_context[Actions::SharedValues::SCAN_GENERATED_PLIST_FILE] if Actions.lane_context[Actions::SharedValues::SCAN_GENERATED_PLIST_FILE]
         params[:path] ||= Actions.lane_context[Actions::SharedValues::SCAN_DERIVED_DATA_PATH] if Actions.lane_context[Actions::SharedValues::SCAN_DERIVED_DATA_PATH]
 
+        params[:silent] = false
```

Comment:
Is there any plan to make this available to be configured by users?

Reply:
Ohhhh, this is a good idea! I should make that an option 😅